### PR TITLE
Feat/8194 deploy to stacks ensono azure subcription

### PR DESCRIPTION
**Pull request by balpurewal**
# Pull Request Template

## 📲 What

Added an infrastructure-aca pipeline to deploy to stacks ensono azure subscription

Added the ability to deploy an app gateway and vnet

Removed frontend app deployment as it shouldn't be here

## 🤔 Why

So we can test the ACA deployment

## 🛠 How

How requests are routed through app gateway to ACA
image

We are reusing some components created from the AKS and Ancillary infrastructure e.g. ACR

### CODE REVIEW

File: build/azDevOps/azure/deploy-infrastructure.yml [hunk 1]
```diff
@@ -179,6 +181,14 @@ stages:
...
                         TF_VAR_stage: ${{ stage.environment_shortname }}
                         TF_VAR_location: $(region)
+                        TF_VAR_create_key_vault: $(create_key_vault)
+                        TF_VAR_create_acr: $(create_acr)
+                        TF_VAR_acr_resource_group: $(acr_resource_group)
+                        TF_VAR_acr_name: $(acr_name)
+                        TF_VAR_create_app_insights: $(create_app_insights)
+                        TF_VAR_dns_resource_group: $(dns_resource_group)
+                        TF_VAR_dns_zone: ${{ stage.dns_zone }}
+                        TF_VAR_create_acavnet: $(create_acavnet)
```
Comment (RichardSlater) [inline build/azDevOps/azure/deploy-infrastructure.yml:191]: My only comment is should this be `TF_VAR_create_aca_vnet` for consistency?

Comment (balpurewal) [reply build/azDevOps/azure/deploy-infrastructure.yml:191]: thanks, still need to make quite a few more changes. I should of created a draft PR. Thanks for taking an early look though

**Comment by RichardSlater** [comment]
@balpurewal can I ask you to incorporate an upgrade to [`cookie` 0.7.0](https://github.com/Ensono/stacks-infrastructure-aca/security/dependabot/10) to mitigate CVE-2024-47764
 as part of this change?

**Comment by balpurewal** [comment]
@RichardSlater yeah sure, that sample app shouldn't be part of this project anyway

**Comment by RichardSlater** [comment]
> @RichardSlater yeah sure, that sample app shouldn't be part of this project anyway

Removing it would also be a solution then :)

**Review by saulfrance — APPROVED** [review]

**Pull request merged**